dbms/src/Server/Server.cpp

```diff
@@ -1566,8 +1566,10 @@ int Server::main(const std::vector<std::string> & /*args*/)
 #ifdef DBMS_PUBLIC_GTEST
     LocalAdmissionController::global_instance = std::make_unique<MockLocalAdmissionController>();
 #else
-    LocalAdmissionController::global_instance
-        = std::make_unique<LocalAdmissionController>(tmt_context.getKVCluster(), tmt_context.getEtcdClient());
+    LocalAdmissionController::global_instance = std::make_unique<LocalAdmissionController>(
+        tmt_context.getKVCluster(),
+        tmt_context.getEtcdClient(),
+        global_context->getSettingsRef().enable_pipeline);
```

Comment:
But it is need to restart TiFlash to enable the pipeline.
So there may be enable pipeline but disable resource control

Reply:
I think no need to support enable pipeline but disable resource control, just as we talked with PM.
PS: should change config name from `enable_pipeline` to `enable_resource_control` next pr.